ops/jenkins/bfd-jenkins-sonarqube/Jenkinsfile

```diff
@@ -1,8 +1,12 @@
 #!/usr/bin/env groovy
 
 // These variables are accessible throughout this file (except inside methods and classes).
-def verboseMaven = params.verbose_mvn_logging
-def appBuildResults
+def verboseMaven
```

Comment:
I *think* `null` is a falsey value in groovy already, so this shouldn't be necessary.

Reply:
> I _think_ `null` is a falsey value in groovy already, so this shouldn't be necessary.

I'm no expert, I just know that assigning variables to e.g. `params.foo` **always** fails on the first run in Jenkins. This _trashy_ little solution helps us avoid the first-run failure that impacted this and other jobs. Have you seen this similar behavior in e.g. [bfd-build-apps](https://jenkins-east.cloud.cms.gov/bfd/view/Build/job/bfd-build-apps/)?

Reply:
Ope, yes. Disregard then :)